src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java

```diff
@@ -611,29 +451,29 @@ && isPartyProvidedWithEmail(party.getValue()) && hasDashboardAccess(party)) {
 
     private void sendNotificationsToParty(CaseData caseData,
                                           Element<PartyDetails> party,
-                                          String authorisation,
                                           Map<String, Object> dynamicDataForEmail,
                                           List<Document> orderDocuments,
                                           List<Element<BulkPrintOrderDetail>> bulkPrintOrderDetails,
-                                          SendgridEmailTemplateNames sengGridTemplate) {
+                                          SendgridEmailTemplateNames sengGridTemplate,
+                                          EmailTemplateNames emailTemplate) {
         log.debug("=== Party contact preference ==== {}", party.getValue().getContactPreferences());
         if (ContactPreferences.email.equals(party.getValue().getContactPreferences())
             && isPartyProvidedWithEmail(party.getValue())) {
             dynamicDataForEmail.put("name", party.getValue().getLabelForDynamicList());
             dynamicDataForEmail.put(DASH_BOARD_LINK, citizenDashboardUrl);
 
-            if (hasDashboardAccess(party)) {
```

Comment:
Why this change? Whats the need of adding new method with different name when logic is still the same.

Reply:
The classic method hasAccesscode was duplicated It was introduced with a new method name by james so reverted that and kept it same as SOA